tensorflow/core/kernels/segment_reduction_ops_gpu.cu.h

```diff
@@ -106,7 +108,7 @@ __global__ void SortedSegmentReductionCustomKernel(
     Index last_output_segment_id = output_outer_dim_size;
 
     const Index actual_stripe_height =
-        min(Index(OuterDimTileSize),
+        cuda::std::min(Index(OuterDimTileSize),
```

Comment:
This will break ADL.  It should probably be something like
```
using cuda::std::min;
min(Index(OuterDimTileSize));
```
but again, `std::min` should probably work here rather than `cuda::std::min`.